public/app/plugins/datasource/loki/components/LokiLabelBrowser.tsx

```diff
@@ -467,7 +466,12 @@ export class UnthemedLokiLabelBrowser extends React.Component<BrowserProps, Brow
             2. Find values for the selected labels
           </Label>
           <div>
-            <Input onChange={this.onChangeSearch} aria-label="Filter expression for values" value={searchTerm} />
+            <Input
+              onChange={this.onChangeSearch}
+              aria-label="Filter expression for values"
+              value={searchTerm}
+              placeholder={'Enter a label name'}
```

Comment:
I misinterpreted the explanation of this field, so `Enter a label value` was correct. Feel free to apply the suggestion from here.

```suggestion
              placeholder={'Enter a label value'}
```

![imagen](https://user-images.githubusercontent.com/1069378/203571103-c755d939-a16d-471f-9a97-eb37cefc13d7.png)